lib/local_authorities_slug_importer.rb

```diff
@@ -0,0 +1,44 @@
+require 'json_downloader'
+
+class LocalAuthoritiesSlugImporter
+  include JsonDownloader
+
+  AUTHORITIES_URL = "https://raw.githubusercontent.com/alphagov/frontend/master/lib/data/authorities.json"
+
+  def self.import_slugs
+    new.import
+  end
+
+  def import
+    LocalAuthority.find_each do |authority|
```

Comment:
[`find_each` defaults to batches of 1000](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each), so unless we override that we aren't getting the benefit of using it here.